python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -84,9 +204,7 @@ def _on_before_paint(self, widget, model_index, style_options):
         # v004 (2014-02-21 12:34)
 
         header_str = ""
-
-        if sg_item.get("version_number"):
-            header_str += "<b style='color:#2C93E2'>Version %03d</b>" % sg_item.get("version_number")
+        header_str += "<b style='color:#2C93E2'>Version %03d</b>" % (sg_item.get("version_number") or 0)
```

Comment:
It's the shotgun API basically - you ask it "find shots for me and return values name, in_frame and description" - and the data it returns is None for all those fields where no value has been set; 

```
{"type": "Shot", "id": 1234, "description": "Foo bar", "in_frame": None }
```